src/status_im/contexts/syncing/device/view.cljs

```diff
@@ -1,17 +1,47 @@
 (ns status-im.contexts.syncing.device.view
   (:require
     [quo.core :as quo]
+    [react-native.core :as rn]
     [status-im.contexts.syncing.device.style :as style]
-    [utils.i18n :as i18n]))
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn- hide-sheet
+  []
+  (rf/dispatch [:hide-bottom-sheet]))
+
+(defn- unpair-confirmation
+  [installation-id name]
+  (fn []
```

Comment:
We don't use `(fn [] ...)` anymore.
CC: @flexsurfer

Reply:
This is a bit different because this function isn't used as a component but called to return content/component of the bottom-sheet